packages/appium/docs/en/index.md

```diff
@@ -1,9 +1,61 @@
 ---
-title: Home
+title: Appium Documentation
 ---
 
 # Appium Documentation
 
-This is the Appium documentation.
+<div style="text-align: center">
+  <img src="assets/images/appium-logo-horiz.png" style="max-width: 400px;" />
+</div>
 
-TODO
+Welcome to the Appium documentation! Appium is an open-source project and ecosystem of related
+software, designed to facilitate UI automation of many app platforms, including mobile (iOS,
+Android, Tizen), browser (Chrome, Firefox, Safari), desktop (macOS, Windows), TV (Roku, tvOS,
```

Comment:
Maybe the Tizen can be in TV rather than mobile nowadays

Reply:
i think the samsung tizen driver doesn't work on the tv, only on their mobile platform

Reply:
but maybe the tizen mobile driver doesn't work at all in which case yeah we shouldn't mention it